pages/activities.jsx

```diff
@@ -277,7 +277,7 @@ var ActivitiesPage = React.createClass({
               <HiveLink
                 src1x="/img/pages/activities/svg/logo-hive-community.svg"
```

Comment:
If we own the `HiveLink` component, it would make more sense to then require passing in an object of sources:
```
<HiveLink src={{ base: "...", x2: "...", x4: "..." }} ...../>
```
Then that way you are forced to think of what kind of data you're putting in, since ultimately it's *rendered* as an `<img>` tag, which doesn't have a "srcx1" attribute
```